swri_transform_util/test/initialize_origin_manual.test.py

```diff
@@ -63,13 +63,25 @@ def generate_test_description():
         parameters=[{
             "local_xy_frame": "/far_field",
             "local_xy_origin": "swri",
-            "local_xy_origins": [29.45196669, -98.61370577, 233.719, 0.0],
+            "local_xy_origins": [29.45196669, -98.61370577, 233.719],
+        }]
+    )
+
+    init_origin2 = Node(
+        package="swri_transform_util",
+        name="origin",
+        executable="initialize_origin.py",
+        parameters=[{
+            "local_xy_frame": "/far_field",
```

Comment:
Should /far_field be changed to /map?

Reply:
We might need to update our test cases, but I think this should be /map.

Reply:
It's entirely cosmetic as far as I know. Do you want this change here or in another issue/PR?